kakaemma/DecodeString
# #181166386 Setup Rspec and Tests for the challenge

#### What does this PR do?
This sets up Rspec and tests for the challenge
#### Description of Task to be completed?
- Add Rspec
- Write tests for the challenge
#### How should this be manually tested?
- Pull and checkout to this branch
- Swich to the challenge folder
- run `bundle install` on your command line
- run `rspec` or `bundle exec rspec` on your command line
Three tests should be failing and one test passing
###  What are the relevant pivotal tracker stories?
[#181166386](https://www.pivotaltracker.com/story/show/181166386)
#### Design prototype link(s) (for UI stories)
N/A
#### Implementation Plan link(s)
N/A
#### Any background context you want to add?
N/A
#### Screenshots
N/A
